include/swift/SIL/Projection.h

```diff
@@ -289,6 +289,8 @@ class Projection {
 
   /// Apply this projection to \p BaseType and return the relevant subfield's
   /// SILType if BaseField has less subtypes than projection's offset.
+  ///
+  /// WARNING: This is not a constant time operation.
```

Comment:
Can you add here that it is not a constant time operation until we fix the typechecker? I am scared that this comment will be missed when the type checker is fixed. With that comment, it will be obvious that the comment is out of date.

Reply:
No, the implementation of this method itself isn't constant-time, regardless of the type checker's implementation. Code that needs to scale should find a way not to call it.

Reply:
From talking with Andy offline, the reason he said that this is not constant time is b/c of getVarDecl requesting all of the stored properties. I asked him to include that explicitly in this warning.